### ISSUE TITLE
duplicate-except throws red-herring

### ISSUE THREAD

**Issue by gbataille** [PyCQA/pylint#1050]
Hi guys, not sure this is for you or the pylint-django guys

### Steps to reproduce
1. in a django project with 2 models
2. create a piece of code like this
```
    try:
        print("foo")
    except Model1.DoesNotExist:
        print("bar")
    except Model2.DoesNotExist:
        print("baz")
```

### Current behavior
This raises a W0705 duplicate-except saying it's catching django.core.exceptions.ObjectDoesNotExist. both those exception (as far as I understand it) are indeed inheriting from ObjectDoesNotExist (common ancestor), but they are not the same class

### Expected behavior
Do not raise this issue on common ancestor

### pylint --version output
pylint 1.5.6,
astroid 1.4.6
Python 2.7.12 (default, Jun 29 2016, 14:05:02)
[GCC 4.2.1 Compatible Apple LLVM 7.3.0 (clang-703.0.31)]

**Comment by PCManticore** [comment]
Thank you, this is probably on our side.

**Comment by PCManticore** [comment]
I think this is probably on pylint-django's side. Pylint already handles the case where the exceptions are in fact different, since it does the check by identity rather than name. In this case though, pylint_django has a transform for Model.DoesNotExist which returns always the same object. It should be changed to return different instances ( https://github.com/landscapeio/pylint-django/blob/master/pylint_django/transforms/transforms/django_db_models.py#L11 ) Can you report it to them and let me know how it goes?

**Comment by gbataille** [comment]
for reference https://github.com/landscapeio/pylint-django/issues/81
